schema/sh_voices.lua

```diff
@@ -296,7 +294,7 @@ Schema.voices.Add("Combine", "SEARCH", "Search!", "npc/metropolice/vo/search.wav
 Schema.voices.Add("Combine", "SEARCHING FOR SUSPECT", "Searching for suspect.", "npc/metropolice/vo/searchingforsuspect.wav")
 Schema.voices.Add("Combine", "SECOND WARNING", "This is your second warning!", "npc/metropolice/vo/thisisyoursecondwarning.wav")
 Schema.voices.Add("Combine", "SECTOR NOT STERILE", "Confirmed- sector not sterile.", "npc/combine_soldier/vo/confirmsectornotsterile.wav")
-Schema.voices.Add("Combine", "SECTOR SECURE", "Sector is now secure.", "npc/combine_soldier/vo/sectorisnotsecure.wav")
+Schema.voices.Add("Combine", "SECTOR NOT SECURE", "Sector is not secure.", "npc/combine_soldier/vo/sectorisnotsecure.wav")
```

Comment:
I've listened to it ingame and it's supposed to be "not". 
Reasons for this:
npc/combine_soldier/vo/sectoris**not**secure.wav
and
in German it is also "not"